### PR TITLE
DDW Enhancements and MS4 Permit Layer

### PR DESCRIPTION
**Pull request by stdavis**
Both Rachel Albritton and Mark Stanger have approved of these additions to be pushed to production.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "deq-enviro",
-  "version": "1.7.4",
+  "version": "1.8.0-1",
```
Comment (steveoh) [inline bower.json:3]:
```suggestion
  "version": "1.8.0",
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "deq-enviro",
-  "version": "1.7.4",
+  "version": "1.8.0-1",
```
Comment (steveoh) [inline package.json:3]:
```suggestion
  "version": "1.8.0",
```

**Review by steveoh — COMMENTED** [review]
looks good to me

File: _src/app/config.js [hunk 3]
```diff
@@ -92,7 +92,7 @@ define([
 
         // version.: String
         //      The version number.
-        version: '1.7.4',
+        version: '1.8.0-1',
```
[the remark this answers is not in the document]
Comment (stdavis) [reply _src/app/config.js:95]: I plan on bumping this in `main` after it's merged.

File: _src/app/search/IdentifyPane.js [hunk 4]
```diff
@@ -87,12 +87,19 @@ define([
...
+            var urlRegex = /^https?:\/\/\S*$/;
             var columns = [
                 {
                     field: 'fieldAlias'
                 }, {
                     field: 'value',
-                    formatter: item => item
+                    formatter: item => {
+                        if (urlRegex.test(item)) {
+                            return `<a href="${item}" target="_blank">link</a>`;
```
[the remark this answers is not in the document]
Comment (stdavis) [reply _src/app/search/IdentifyPane.js:98]: This isn't necessary if I set `target="_blank"` is it? The [MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener) makes me think so.

[the remark this answers is not in the document]
Comment (steveoh) [reply _src/app/search/IdentifyPane.js:98]: that looks new to me. i guess not?

**Pull request merged**